src/features/github/GitHubUtils.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Exchanges the Access Code with the back-end to receive an Auth-Token
+ *
+ * @param {string} backendLink The address where the back-end microservice is deployed
+ * @param {string} messageBody The message body. Must be URL-encoded
+ * @return {Promise<Response>} A promise for a HTML response with an 'auth_token' field
+ */
+export async function exchangeAccessCodeForAuthTokenContainingObject(
+  backendLink: string,
+  messageBody: string
+): Promise<Response> {
+  return await fetch(backendLink, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
```

Comment:
Please add a space after the semicolon (sorry, just nitpicking...)

Reply:
The previews have been removed here: 3f4145e.
Added a space here: 8a18ab9.

Reply:
Thanks a lot for taking the time to review my changes! I don't mind nitpicking at all.